src/chrome/content/rules/spnet.ru.xml

```diff
@@ -1,24 +1,257 @@
 <!--
-spnet.ru mixed content
-www.spnet.ru mismatch
-okna.spnet.ru mismatch
-r3u25.spnet.ru mismatch
-www.school14.spnet.ru mismatch
-4x4drive.ru.spnet.ru protocol error
-csit.spnet.ru mismatch
+01045.spnet.ru ¹
+100rub.spnet.ru ¹
+12345.spnet.ru ¹
+12688.spnet.ru ¹
+1380stl5.spnet.ru ³
+185-132-240-0.spnet.ru ¹
+185-132-240-1.spnet.ru ¹
+185-132-240-2.spnet.ru ¹
+185-132-240-3.spnet.ru ¹
+185-132-240-4.spnet.ru ¹
+185-132-240-5.spnet.ru ¹
+185-132-240-6.spnet.ru ¹
+185-132-240-7.spnet.ru ¹
+185-132-240-8.spnet.ru ¹
+185-132-240-9.spnet.ru ¹
+185-132-241-0.spnet.ru ¹
+185-132-241-1.spnet.ru ¹
+185-132-241-2.spnet.ru ¹
+185-132-241-3.spnet.ru ¹
+185-132-241-4.spnet.ru ¹
+185-132-241-5.spnet.ru ¹
+185-132-241-6.spnet.ru ¹
+185-132-241-7.spnet.ru ¹
+185-132-241-8.spnet.ru ¹
+185-132-241-9.spnet.ru ¹
+185-132-242-0.spnet.ru ¹
+185-132-242-1.spnet.ru ¹
+185-132-242-4.spnet.ru ¹
+185-132-242-7.spnet.ru ¹
+185-132-242-9.spnet.ru ¹
+185-132-243-0.spnet.ru ¹
+185-132-243-1.spnet.ru ¹
+185-132-243-2.spnet.ru ¹
+185-132-243-3.spnet.ru ¹
+185-132-243-4.spnet.ru ¹
+185-132-243-5.spnet.ru ¹
+185-132-243-6.spnet.ru ¹
+185-132-243-7.spnet.ru ¹
+185-132-243-8.spnet.ru ¹
+185-132-243-9.spnet.ru ¹
+188-35-10-0.spnet.ru ¹
+188-35-10-1.spnet.ru ¹
+188-35-10-2.spnet.ru ¹
+188-35-10-3.spnet.ru ¹
+188-35-10-4.spnet.ru ¹
+188-35-10-5.spnet.ru ¹
+188-35-10-6.spnet.ru ¹
+188-35-10-7.spnet.ru ¹
+188-35-10-8.spnet.ru ¹
+188-35-10-9.spnet.ru ¹
+188-35-11-0.spnet.ru ¹
+188-35-11-1.spnet.ru ¹
+188-35-11-2.spnet.ru ¹
+188-35-11-3.spnet.ru ¹
+188-35-11-4.spnet.ru ¹
+188-35-11-5.spnet.ru ¹
+188-35-11-6.spnet.ru ¹
+188-35-11-7.spnet.ru ¹
+188-35-11-8.spnet.ru ¹
+188-35-11-9.spnet.ru ¹
+188-35-8-0.spnet.ru ¹
+188-35-8-1.spnet.ru ¹
+188-35-8-2.spnet.ru ¹
+188-35-8-3.spnet.ru ¹
+188-35-8-4.spnet.ru ¹
+188-35-8-5.spnet.ru ¹
+188-35-8-6.spnet.ru ¹
+188-35-8-7.spnet.ru ¹
+188-35-8-8.spnet.ru ¹
+188-35-8-9.spnet.ru ¹
+188-35-9-0.spnet.ru ¹
+188-35-9-1.spnet.ru ¹
+188-35-9-2.spnet.ru ¹
+188-35-9-3.spnet.ru ¹
+188-35-9-4.spnet.ru ¹
+188-35-9-5.spnet.ru ¹
+188-35-9-6.spnet.ru ¹
+188-35-9-7.spnet.ru ¹
+188-35-9-8.spnet.ru ¹
+188-35-9-9.spnet.ru ¹
+1997.spnet.ru ¹
+20101980.spnet.ru ¹
+2129977.spnet.ru ¹
+23072001.spnet.ru ¹
+2garin.spnet.ru ¹
+2z.spnet.ru ¹
+6narvick6.spnet.ru ¹
+7243311.spnet.ru ¹
+82.spnet.ru ¹
+846.spnet.ru ¹
+85.spnet.ru ¹
+89-222-212-0.spnet.ru ¹
+89-222-212-1.spnet.ru ¹
+89-222-212-2.spnet.ru ¹
+89-222-212-4.spnet.ru ¹
+89-222-212-5.spnet.ru ¹
+89-222-212-6.spnet.ru ¹
+89-222-212-7.spnet.ru ¹
```

Comment:
For the luck, they are using wildcard domain *.spnet.ru (in addition to any-other-static-domains.spnet.ru), which cert is for hosting.spnet.ru only.